worker/src/http/request.rs

```diff
@@ -131,19 +127,8 @@ pub fn into_wasm(mut req: http::Request<Body>) -> web_sys::Request {
         let _ = r;
     }
 
-    let body = req.into_body();
-    let body = if body.is_none() {
-        None
-    } else {
-        let stream = wasm_streams::ReadableStream::from_stream(body.map(|chunk| {
```

Comment:
Ultimately the bug seems to be in `from_stream`. `from_byte_stream` seems to work though. I'm not sure if that means a different `BoxedBody` that isn't `Bytes` will break?

Reply:
`HttpClone` is only implemented when the body holds `Bytes` so that shouldn't be an issue.